tests/common/gu_utils.py

```diff
@@ -125,6 +125,8 @@ def apply_patch(duthost, json_data, dest_file):
     start_time = time.time()
     output = duthost.shell(cmds, module_ignore_errors=True)
     elapsed_time = time.time() - start_time
+    if duthost.facts['platform'] == 'armhf-nokia_ixs7215_52x-r0':
```

Comment:
Can you create a map around line 18 and define a new function to get hte GCUTIMEOUT?
```
GCUTIMEOUT_MAP { 'default': 600, 'platform_A': 1200, ... }
def get_gcu_timeout(duthost):
```

Otherwise, the platform could fail again if new test cases use the GCUTIMEOUT

Reply:
Added map for GCU_TIMEOUT and function to get value for platforms. Please review change